infullmvp-kotlin/src/main/java/com/infullmobile/android/infullmvp/LaunchableBottomSheet.kt

```diff
@@ -0,0 +1,14 @@
+package com.infullmobile.android.infullmvp
+
+import android.support.v4.app.DialogFragment
+import android.support.v4.app.FragmentManager
+
+open class LaunchableBottomSheet(
```

Comment:
what's this ?

Reply:
We have a `LaunchableIntent` class, so we created lately a `LaunchableBottomSheet`. API wrapper to have nice chains for our navigation stack.

Both of these should rather land in a separate artifact.